src/garage/torch/modules/multi_headed_mlp_module.py

```diff
@@ -118,5 +109,15 @@ def forward(self, input_val):
         x = input_val
         for layer in self._layers:
             x = layer(x)
-
-        return [layer(x) for layer in self._output_layers]
+            if self._hidden_nonlinearity:
```

Comment:
just to be clear, what I'm trying to avoid is having to manually call the non linearities. I think they should just be defined as computations in the `nn.ModuleList`

Reply:
We should have removed the need for this loop, right?